windows-installer/docs/INSTALL.md

```diff
@@ -0,0 +1,87 @@
+# Certbot for Windows (beta)
+
+The Certbot development team is proud to offer you the first beta release of Certbot for Windows.
+
+This document explains how to install Certbot and use it on Windows.
+
+Please note that this option is intended for the situation where your web server runs Windows. If you use Windows on your personal computer but have a web server with a different operating system, you should normally install Certbot on the web server (probably via SSH) instead of on your local computer. This document is not intended to describe that situation.
+
+## Supported features
+
+All usual operations to create and manage an account, manage existing certificates, or select the ACME server, are supported.
+
+In terms of authenticator plugins, `standalone`, `manual` and `webroot` are supported. DNS plugins will be available soon.
+
+For now, no installer plugins are available. `apache` and `nginx` will be available soon, and a new installer for IIS is under development. This means that you will obtain a PEM-formatted certificate and will have to import it into your web server application or keystore by yourself. This may also require you to convert the certificate from PEM format into a different format first.
+
+Automated certificate renewals are supported.
+
+All options available for a given supported command, as described by the documentation, will work in Certbot for Windows.
+
+## Installation
+
+The following operations need to be executed using an account with **administrative privileges** on the machine where you want to install Certbot.
+
+Certbot can be installed through an installer that is available:
+- on GitHub: go to the Certbot GitHub release (https://github.com/certbot/certbot/releases), select the last version of Certbot (v0.40.0 currently), and download the installer named `certbot-beta-installer-win32.exe in the assets.
+- on Certbot's official website: _[specific URL and procedure to be defined]_
```

Comment:
I think we should just suggest one of these two download locations. Ideally, I think we should point people towards eff.org as I think that's where we'll want people to download Certbot in the long term, but I'm not sure if that's ready. I just pinged tech ops asking for an update.

Reply:
Maybe for the beta phase it would make more sense to download it from Github. And this way we do not urge tech ops to deploy a solution for the next release.

Reply:
Yeah let's do that for now. They still haven't gotten back to me.

Reply:
Ok, done.